client/packages/host/src/Admin/SyncSettings.tsx

```diff
@@ -97,16 +97,18 @@ const SyncSettingsForm = ({
         component={
           <NumericTextInput
             value={intervalSeconds}
-            onChange={seconds =>
-              setSettings(
-                'intervalSeconds',
-                NumUtils.constrain(
-                  Math.round(seconds),
-                  1,
-                  Number.MAX_SAFE_INTEGER
-                )
-              )
-            }
+            onChange={seconds => {
+              if (seconds !== undefined) {
+                setSettings(
+                  'intervalSeconds',
+                  NumUtils.constrain(
+                    Math.round(seconds),
+                    1,
+                    Number.MAX_SAFE_INTEGER
+                  )
+                );
+              }
+            }}
```

Comment:
This page doesn't seem to be in use. But again, based on context, it doesn't seem like an empty value should be allowed.